### PR TITLE
50039_book_index_admin

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

**Comment by SSalekin** [comment]
ok

**Comment by Rownaknira** [comment]
ok

File: app/views/admin/books/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<% provide(:title, t(:books_page_title)) %>
```
Comment (dieunb) [inline app/views/admin/books/index.html.erb]: Remove `()`

File: db/seeds.rb [hunk 2]
```diff
@@ -1,4 +1,14 @@
 10.times do |i|
-  User.create! name: Faker::Name.name, email: "user-#{i}@example.com", 
-    role: User.roles[:user], password: "password"
+  User.create! name: Faker::Name.name, email: "user-#{i}@example.com",
```
Comment (dieunb) [inline db/seeds.rb]: Why not use `FactoryGirl` or `Fabrication` gem

File: db/seeds.rb [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+               role: User.roles[:user], password: "password"
+end
+
+Category.all.each do |category|
+  10.times do
+    Book.create title: Faker::Lorem.word,
```
Comment (dieunb) [inline db/seeds.rb]: Same above

File: app/views/admin/books/index.html.erb (hunk 1 above)
Comment (MuhammadTamzid) [inline app/views/admin/books/index.html.erb]: <% provide :title, t(:books_page_title) %>

**Pull request merged**